templates/01_block-scoping.test.js

```diff
@@ -15,14 +15,16 @@ test('can be used in place of `var`', () => {
   expect(isBestBand).toBe(true)
 })
 
-test('can modify the value of a `let` variable', () => {
+test('can modify the value of a `let` variable even in the next block statement', () => {
   // WORKSHOP_START
   // Declare 'releaseName' using 'let', setting the value to 'ES6'
-  // Change value of releaseName to be `ES2015`, the new name for ES6
+  // Change value of releaseName to be `ES2015`, the new name for ES6 wrapped in a block statement
   // WORKSHOP_END
   // FINAL_START
   let releaseName = 'ES6'
+  {
   releaseName = 'ES2015'
+  }
   // FINAL_END
   expect(releaseName).toBe('ES2015')
```

Comment:
What if we change

```
expect(releaseName).toBe('ES2015')
```

To be:

```
expect(releaseName).toBe(/* ENTER YOUR GUESS HERE */)
```

I want to avoid making them change assertions unless we're explicit about it.